include/model_card_manager.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef CARD_MANAGER_HPP_
+#define CARD_MANAGER_HPP_
+#include <memory>
+#include <string>
+#include <vector>
+#include "model_card.hpp"
+
+namespace meow {
+
+struct CardManager {
+private:
+    static inline const std::string filename = "info.json";
```

Comment:
we have a file which contains all of the string constants, please declare this filename there.